app/assets/javascripts/wikis.js

```diff
@@ -15,7 +14,8 @@ function setupWiki(node_id, title, raw, logged_in) {
       preProcessor: preProcessMarkdown,
       postProcessor: postProcessContent,
       extraButtons: {
-        "fa-question": questionForm
+        "fa-question": questionForm,
+        "fa-comment": setupCommentFunction
```

Comment:
Can we make this:

```ruby
<% if params[:inlineCommenting] %>"fa-comment": setupCommentFunction<% end %>
```
? That way we can test this more on production without committing to it just yet. And we can publish the general Rich Wikis inline features without also having to publish inline commenting which may take a little longer?

Reply:
Hi, @aspriya - i think if you can add this flag we can merge and see how this is working. Thanks!